From: Shengjing Zhu <zhsj@debian.org>
Date: Tue, 10 Jan 2023 16:04:11 +0800
Subject: Force UnixTest not to use https_proxy env

---
 test/authority_test.go | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/test/authority_test.go b/test/authority_test.go
index 6cd5d82..0f6b6d5 100644
--- a/test/authority_test.go
+++ b/test/authority_test.go
@@ -25,6 +25,7 @@ import (
 	"testing"
 	"time"
 
+	"google.golang.org/grpc"
 	"google.golang.org/grpc/codes"
 	"google.golang.org/grpc/metadata"
 	"google.golang.org/grpc/status"
@@ -57,7 +58,7 @@ func runUnixTest(t *testing.T, address, target, expectedAuthority string) {
 		address: address,
 		target:  target,
 	}
-	if err := us.Start(nil); err != nil {
+	if err := us.Start(nil, grpc.WithNoProxy()); err != nil {
 		t.Fatalf("Error starting endpoint server: %v", err)
 		return
 	}
